test/Dapper.AOT.Test/Verifiers/DAP241.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Dapper.CodeAnalysis;
+using System.Threading.Tasks;
+using Xunit;
+using Diagnostics = Dapper.CodeAnalysis.DapperAnalyzer.Diagnostics;
+namespace Dapper.AOT.Test.Verifiers;
+
+public class DAP241 : Verifier<DapperAnalyzer>
```

Comment:
IMO this feature warrants VB tests/support too; there should be a `VBVerifyAsync`

Reply:
supported VB!